src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/SetAzureRmAnalysisServicesServer.cs

```diff
@@ -26,37 +27,54 @@ namespace Microsoft.Azure.Commands.AnalysisServices
     [Alias("Set-AzureAs")]
     public class SetAzureAnalysisServicesServer : AnalysisServicesCmdletBase
```

Comment:
@taiwu this cmdlet will now need a default parameter set since you have multiple parameter sets. You could also make `DisableBackup` a mandatory parameter to avoid potential issues with PowerShell determining the set to use, but having a default parameter set is the recommended pattern.